packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/index.js

```diff
@@ -15,14 +15,22 @@ const main = {
   renderRow: props => {
     if (props.mainColumnActiveTab === 'newRecord') {
       return (
-        <div className="ui fluid dina background" style={{ padding: '20px' }}>
+        <div
+          className="ui fluid dina background"
+          style={{ padding: '20px' }}
+        >
           <CreateSpecimen />
         </div>
       )
     }
     return (
-      <div className="ui fluid dina background" style={{ padding: '20px' }}>
-        <ResultTableView />
+      <div
+        className="ui fluid dina background"
+        style={{ padding: '20px'  }}
+      >
+        <ResultTableView
+          availableHeight={props.availableHeight - 100 - 43 - 50}
```

Comment:
good that you calculate the value here and not inside the component, because it's here we specify those values. 

where does 50 come from?